pkg/menu/menu.go

```diff
@@ -162,7 +162,7 @@ func DisplayResult(message []string, uiEvents <-chan ui.Event) (string, error) {
 	p.SetRect(0, 0, wid+2, resultHeight+3)
 	p.TextStyle.Fg = ui.ColorWhite
 
-	hint := "(Press any key to continue, press <Esc> to exit.)"
+	hint := "(Press any key to continue.)"
```

Comment:
I wondered why remove "press <Esc> to exit" here?  <Esc> key in the DisplayResult() is doing different thing from in other two (input box and menu). In DisplayResult() <Esc> key is design for , if the alert message is so long that user have no patient to read that, they may want to use <Esc> to skip this message and immediately return to former window. But press any other key will just move to the next page of the message.

By the way, this is what I planed to do but have no time to: can you add scrollable feature to the message box? it will be very similar to the menu's input. After all pressing a key to turn the page may be not as convinient as just scrolling the wheel.

Reply:
I don't think any of the callers to `DisplayResult()` are checking the return value right now, so I took it out for now. I'm planning to use Prachi's suggestion to return a custom error if the user presses \<Esc>, so I can update `DisplayResult()` with that change also.

Also, good idea with the scrollable message. I'll try to get that change done this week.

Reply:
Sounds great!